README.md

```diff
@@ -1,10 +1,15 @@
-![](http://www.reactiongifs.com/wp-content/uploads/2013/01/elvis-dance.gif)
-
 # elvis [![Build Status](https://travis-ci.org/inaka/elvis.svg?branch=master)](https://travis-ci.org/inaka/elvis)
 
+![Elvis Presley dancing](http://www.reactiongifs.com/wp-content/uploads/2013/01/elvis-dance.gif)
+
 Command-line interface for Elvis, the Erlang style reviewer.
 
+Elvis is a code style checker that can be used to enforce [common guidelines][guidelines] in your Erlang project hosted on GitHub by detecting where these are not being met. Once Elvis is activated for your repository then every time a pull request is created or updated, rules will be run and Elvis will add comments on those lines that break any single one of these rules.
```

Comment:
I think this whole _hosted on GitHub_ is a historical artifact that is no longer true since @erlang-solutions tore down the Elvis server 😢 

We should remove this paragraph.

Reply:
1885cf4